napalm/base/helpers.py

```diff
@@ -61,8 +61,9 @@ def load_template(cls, template_name, template_source=None, template_path=None,
             loader = jinja2.FileSystemLoader(search_path)
             environment = jinja2.Environment(loader=loader)
 
-            for filter_name, filter_function in CustomJinjaFilters.filters().items():
-                environment.filters[filter_name] = filter_function
+            for d in [CustomJinjaFilters.filters(), jinja_filters]:
```

Comment:
Maybe `for filter_name, filter_function in itertools.chain(CustonJinjaFilters.filters().items(), jinja_filters.items()):` ?

Reply:
Ah I see you've uncovered my lack of python experience!